Sources/Runner/main.swift

```diff
@@ -0,0 +1,9 @@
+import PointFree
```

Comment:
Runner is so far just a "test" for if the app can boot.

Reply:
just so I understand, in the future we'd put our "cron" logic in here, like parse arguments passed in from the scheduler to figure out what logic to execute, etc...?

Reply:
Yeah, we can start with a single task that executes, but we'll eventually wanna use a CLI argument parser. Will be fun to play around with an applicative argument printer/parser.